dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -283,6 +283,10 @@ DSRA4014.URL.for.Driver.missing=DSRA4014E: The type java.sql.Driver was specifie
 DSRA4014.URL.for.Driver.missing.explanation=A connection URL must be specified for a data source that is a java.sql.Driver type.
 DSRA4014.URL.for.Driver.missing.useraction=Add a valid URL to the data source.
 
+DSRA4015.no.ucp.connection.pool.datasource=DSRA4015E: The data source {0} was configured with the ConnectionPoolDataSource type, which is not supported by Oracle UCP. Instead use the XADataSource or DataSource type.
```

Comment:
Suggestion:
The {0} data source was configured with the ConnectionPoolDataSource type, which is not supported by Oracle UCP. Instead, use the XADataSource or DataSource type.